src/index.ts

```diff
@@ -234,9 +234,15 @@ function teenyRequest(reqOpts: r.Options, callback?: r.RequestCallback) {
 }
 
 teenyRequest.defaults = (defaults: r.Options) => {
-  return (reqOpts: r.Options, callback: r.RequestCallback) => {
-    return teenyRequest({...defaults, ...reqOpts}, callback);
-  };
+  return (reqOpts: r.Options, callback?: r.RequestCallback): PassThrough|
+      void => {
+        const opts = {...defaults, ...reqOpts};
+        if (callback === undefined) {
+          return teenyRequest(opts);
+        } else {
```

Comment:
You can drop the else here since it returns above :)

Reply:
fixed :)